src/main/java/core/basesyntax/reader/CsvReaderImpl.java

```
@@ -0,0 +1,21 @@
+package core.basesyntax.reader;
+
+import java.io.*;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.util.Collections;
+import java.util.List;
+
+public class CSVReaderImpl implements CSVReader {
+
+    @Override
+    public List<String> read(String pathFile) {
+        try {
+            return Files.readAllLines(Paths.get(pathFile));
+        } catch (IOException e) {
+            System.out.println("Cannot read file:" + pathFile);
+            e.printStackTrace();
+            return Collections.emptyList();
```

This violates checklist item #6: "Don't ignore exceptions in try-catch blocks. Don't leave `e.printStackTrace()` or `System.out.println(e.getMessage())` in the catch blocks. Let's rethrow a RuntimeException with an **informative** message and exception object." Instead of printing the error and returning an empty list, you should re-throw a `RuntimeException` that wraps the original `IOException`.